README.md

```diff
@@ -102,9 +102,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration, resources/assets used
-- etc
+- [The Odin Project](https://www.theodinproject.com/dashboard)
```

Comment:
the link can be 
https://www.theodinproject.com/lessons/foundations-rock-paper-scissors
instead